lib/container/container.go

```diff
@@ -0,0 +1,93 @@
+// package container contains convenience functions for manipulating containers
+package container
+
+import (
+	"io"
+	"log"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+func RawStartContainer(opts docker.CreateContainerOptions) (string, error) {
+	client, err := docker.NewClient("unix:///var/run/docker.sock")
```

Comment:
all the hardcoding of the DOCKER_HOST makes this untestable on other OS's, or remotely